yamale/validators/validators.py

```diff
@@ -166,6 +167,40 @@ def get_name(self):
         return self.regex_name or self.tag + " match"
 
 
+class Ip(Validator):
+    """IP address validator"""
+    tag = 'ip'
+    constraints = [con.IpVersion]
+
+    def _is_valid(self, value):
+        return self.ip_address(value)
+
+    def py2_ip_address(self, value):
+        try:
+            ipaddress.ip_interface(unicode(value))
+        except Exception:
+            return False
+        return True
+
+    def ip_address(self, value):
+        try:
```

Comment:
You can put the `import ipaddress` statement here, in the function. Python caches imports so re-importing has littte/no performance impact.

Reply:
Ah, yeah, I misunderstood previously.  Thought you meant at the top, but can definitely do in the function.  Thanks!